index.rst

```diff
@@ -1,62 +1,147 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+.. TODO: Delete the note below before merging new content to the main branch.
 
-  Sections
-  ========
+.. note::
 
-  and
+   **This technote is not yet published.**
 
-  Subsections
-  -----------
+   The description of the Campaign Management system in the first draft of DMTN-181 :cite:`DMTN-181` implies a few butler/pipeline middleware feature requests, and the processes and tooling for defining campaigns are expected to need new middleware support as well.
+   This technote will attempt to aggregate those feature requests, while outlining a few possible approaches for how campaign definition and management will interact with the middleware, based on the absence or presence of potential new middleware functionality.
 
-  and
+Introduction and Scope
+======================
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+DMTN-181 :cite:`DMTN-181` describes a Campaign Management system that provides tooling for organizing and tracking the multiple workflows that comprise a pipeline processing campaign.
+It also touches on what we will here call (for lack of a better term, so far) *Campaign Definition*: the tools and processes for determining the datasets, software, and configuration that will go into a campaign.
+The datasets are our primary concern here, and for these :cite:`DMTN-181` focuses on the concept of exposure *exclusion lists* represented as *data ID sets*.
```

Comment:
There's also how this fits in with summit users creating tags and quality flags for datasets (see also DMTN-204) and where that information ends up in USDF.